### DISCUSSION TITLE
Single wire-splitter can't have multiple outputs?

### DISCUSSION THREAD

**Discussion by mengstr** [hneemann/Digital#1189] [General]
I was trying to use a splitter as a convenient and compact way for hooking up the decimal points of a couple of 7-segment displays.

The dox says "_For example, "4-7,0-3" configures bits 4-7 and 0-3. This notation allows any bit arrangement. Output bits can also be output several times: "0-7,1-6,4-7"._" which works just fine.  But just a single input bit and replicating that onto several outputs fails.

<img width="660" alt="splitter" src="https://github.com/hneemann/Digital/assets/325326/2150db2f-b71b-422f-823e-e383b72f1aeb">

**Comment by hneemann** [comment]
Why not connecting all the pins directly with a simple wire?

**Comment by mengstr** [comment]
I ended up doing that.  It was mostly a matter of aesthetics since I already had a a splitter there and they would nicely overlap and they are not flashing and changing colors according to the logic level either. ;-)

<img width="976" alt="splittr" src="https://github.com/hneemann/Digital/assets/325326/4f135321-3629-42bb-b662-52b02077c36f">
